### ISSUE TITLE
[Quest] [NPC] The Nightmare's Corruption - Twilight Corrupter 

### ISSUE THREAD

**Issue by Lath76** [dalaranwow/dalaran-wow#5678]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The Twilight Corrupter is located in the Twilight Grove in Darkshore, you need to kill him as part of the quest The Nightmare's Corruption. When solo fighting him, i get mind controled and he resets. As i said i'm doing this solo (no raid, party, pets or anything, just me) it should not be possible to get mind controled since there is only one target in his aggro table.

**Expected behaviour**: Tell us what should happen instead
Twilight Corrutor should not mind control when there is only one target in his aggro table.

**Steps to reproduce the problem**:

1. Have the quest The Nightmare's Corruption and go to the Twilight Grove in Darkshore
2. Find Twilight Corruptor and engage him alone (dont even use pets)
3. Fight him and you'll get mind controled (happens to me at around 1 minute fight)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
